Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:8.5-stretch
+LABEL maintainer="will.price94@gmail.com"
+LABEL version="0.0.1"
+
+RUN apt-get update -qq && \
+    apt-get install -y python
+
+
+ENV HOME=/app
+WORKDIR $HOME
+RUN mkdir -p $HOME
+RUN chown node:node $HOME
+
+USER node
+COPY package.json package-lock.json $HOME/
+RUN npm install
+COPY . $HOME
+RUN npm run build
+RUN npm run typecheck
+RUN npm test
+
+ENV DB_DIR=$HOME/db
+ENV VIMFLOWY_PASSWORD=vimflowy123
```

Comment:
are you planning on mounting DB_DIR in from the host filesystem?  I think it would make sense to do that

Reply:
Yep that's correct, it'll be a volume (which facilitates a user mounting an external dir into the container, but doesn't require it)